P10AwsServerlessStack/src/pkg/user/user.go

```diff
@@ -0,0 +1,148 @@
+package user
+
+import (
+	"aws-serverless-stack/src/pkg/validators"
+	"encoding/json"
+	"errors"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+)
+
+var (
+	ErrorFailedToFetchRecord     = "failed to fetch record"
+	ErrorFailedToUnmarshalRecord = "failed to unmarshal record"
+	ErrorInvalidUserData         = "invalid user data"
+	ErrorInvalidEmail            = "invalid email"
+	ErrorCouldNotMarshalItem     = "could not marshal item"
+	ErrorCouldNotDeleteItem      = "could not delete item"
+	ErrorCouldNotDynamoPutItem   = "could not dynamo put item"
+	ErrorUserAlreadyExists       = "user.User already exists"
+	ErrorUserDoesNotExist        = "user.User does not exist"
+)
+
+type User struct {
+	Email     string `json:"email"`
+	FirstName string `json:"firstName"`
+	LastName  string `json:"lastName"`
```

Comment:
Please add 3 more fields like Date of birth and maritalStatus and is Employed

Reply:
fixed